.travis.yml

```diff
@@ -66,12 +66,6 @@ matrix:
     - go: 1.12.x
       os: linux
       env: GO111MODULE=on GOFLAGS=
-    - go: 1.12.x
-      os: osx
-      env: GO111MODULE=off GOFLAGS=
```

Comment:
In removing this config we don't test osx without modules any more - I would recommend setting these flags onto the 1.13 version